README.md

```diff
@@ -81,6 +82,10 @@ export default class App extends Component {
     const search = (tag, input) => {
       tag.label.toLowerCase().includes(input.toLowerCase())
     }
+    
+    const exactMatch = (tag, input) => {
```

Comment:
Even though the table above states that `search` and `exactMatch` are not required, I think that having them in the `Example` section makes it seem like they are required. Can we add a comment above both of these stating that they're optional and showing what the default function does?